tests/conftest.py

```diff
@@ -26,3 +29,22 @@ def flask_test_client():
     """
     with create_app().test_client() as test_client:
         yield test_client
+
+
+@pytest.fixture
+def seed_test_dataset(flask_test_client):
+    """
+    Pushes an application context to a test.
+
+    :return: a test application context.
+    """
+    with flask_test_client.application.app_context():
+        seed_test_database(Organisation, db_seed.ORGANISATION_DATA)
+        seed_test_database(Contact, db_seed.CONTACT_DATA)
+        seed_test_database(Package, db_seed.PACKAGE_DATA)
```

Comment:
We also need to add seed data for the other 13 or so tables. If it's easier and you feel the ticket is getting too big we could get this tidied up, and merge in what we have, then add the remaining models in an additional feature branch?

Will need to rebase in / resolve conflicts for the fix for FK constraints that is in PR: https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/31 for this to work properly